.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
```

Comment:
hm generally we don't want to use npm and yarn as package installers in the same codebase because using them simultaneously leads to dependency consistency conflicts, so if possible can we remove package-lock.json (what npm uses to keep track of package dependencies) and reference little-state-machine using `yarn little-state-machine`? I'm not too familiar with this library so if there are limitations to how we can use it let me know

Reply:
oh yeah i definitely meant to yarn add not npm install :') i'll delete this line!